src/components/families/family-form/family-parent-fields/FamilyParentFields.tsx

```diff
@@ -44,11 +42,7 @@ const FamilyParentFields = ({ dynamicFields, family, onChange }: Props) => (
           },
         });
       }}
-      value={
-        family.parent.date_of_birth
-          ? moment(family.parent.date_of_birth).format("YYYY-MM-DD")
-          : ""
-      }
+      value={family.parent.date_of_birth || ""}
```

Comment:
wait, is there a reason we don't map the default fields in a similar way to the dynamic fields? I know this isn't part of the PR, but since you're refactoring this stuff anyway, I thought I'd bring it up.

Reply:
this is because the dynamic fields work with the information json object, so its [type](https://github.com/uwblueprint/project-read-frontend/blob/2bc760e1470afb9cade9d7a9ab23a3a96e4da415/src/types/index.ts#L25) is more loosely defined (just a key/value pair)! the default fields are explicit properties on the family type, so we enforce better type safety accessing them this way

not sure if there's a way to loop through them more concisely, but i don't mind how explicit this is atm!